sunkit_spex/extern/stix.py

```diff
@@ -0,0 +1,1393 @@
+import warnings
+
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy.time import Time, TimeDelta
+from astropy import units as u
+from astropy.io import fits
+
+from sunkit_spex.legacy.fitting_legacy import instruments
+
+
+class StixLoader(instruments.InstrumentBlueprint):
```

Comment:
camel case for classes, snake case for functions, variable etc at 99% of the time
```suggestion
class STIXLoader(instruments.InstrumentBlueprint):
```